java/ql/src/Security/CWE/CWE-074/XsltInjection.ql

```diff
@@ -11,8 +11,7 @@
  */
 
 import java
-import semmle.code.java.dataflow.FlowSources
-import XsltInjectionLib
+import semmle.code.java.security.XsltInjectionQuery
```

Comment:
Comment for line 3 above (as I can't comment on the line itself):
Suggesting:
`Performing an XSLT transformation with user-controlled stylesheets can lead to
information disclosure or execution of arbitrary code.`

Reply:
Don't worry about directly committing to the branch, It's totally fine :-)

Thank you for the review! I applied your suggestion regarding the query description.